ui/component/page/view.jsx

```diff
@@ -13,18 +13,19 @@ type Props = {
   isUpgradeAvailable: boolean,
   authPage: boolean,
   authenticated: boolean,
+  noHeader: boolean,
 };
 
 function Page(props: Props) {
-  const { children, className, authPage = false, authenticated } = props;
+  const { children, className, authPage = false, authenticated, noHeader } = props;
   const obscureSideNavigation = IS_WEB ? !authenticated : false;
 
   return (
     <Fragment>
-      <Header authHeader={authPage} />
+      {!noHeader && <Header authHeader={authPage} />}
```

Comment:
I think we should still keep the header so the LBRY logo is visible. I would just pass `authPage` to the `<Page />` component. (and possibly rename that value.

Reply:
Fine - my goal was to get rid of the [X]. I'll revisit.

Reply:
Calling it fine for now.